MetaMask/metamask-mobile
# feat: add favorites to browser menu

## **Description**



We're adding in a "Go to Favorites" option into the browser menu so users can navigate to https://home.metamask.io, allowing them to access their previous favorites pages. After switching to the Portfolio dapps for a user's homepage, we received a decent amount of user feedback wanting this feature back in.

## **Related issues**

Fixes:

## **Manual testing steps**

1. Open the in-app browser
2. Navigate to any website that's not https://home.metamask.io
3. Open the browser menu by tapping on the "more" icon
4. See the new "Go to Favorites" option, tap that.
5. Confirm that the user is taken to https://home.metamask.io

## **Screenshots/Recordings**



### **Before**

![IMG_8936](https://github.com/user-attachments/assets/36f65f0c-7929-4058-98ba-5800aec70be5)

### **After**

![Screenshot 2024-10-28 at 11 13 04 AM](https://github.com/user-attachments/assets/68a2898e-2a72-43ee-8572-155a7b5a2028)

Actual

https://github.com/user-attachments/assets/e6aabfd6-3746-4075-ad47-b8140afbc3fd





## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
